beefy-test/src/client_tests.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
```

Comment:
is this the exact same test that it used to run from `client.rs` but now it runs on this dedicated test file?

Reply:
Yep, moved the unit tests into a separate file.